fai/fai.go

```diff
@@ -57,6 +57,9 @@ func NewIndex(fasta io.Reader) (Index, error) {
 		if len(b) == 0 {
 			continue
 		}
+		if bytes.Equal(bytes.TrimRight(b, " "), []byte{'>'}) {
```

Comment:
`b` has been space-trimmed, so it should be enough to compare to the ">" literal.

Reply:
Resolved in 25658fd85b0583d425debc45806b89a25d9c61b0